### ISSUE TITLE
Auto-sizing of Width and Height is not working when size other than btn-md is used

### ISSUE THREAD

**Issue by sstragier** [ziscloud/angular-bootstrap-toggle#25]
When using size with the toggle, the width and height that are auto calculated are for btn-md. It seems like the issue is because the computeStyle function is being called before the size attributes are applied to the DOM.

Here is sample code to reproduce:
```
<toggle ng-model="model" size="btn-xs"></toggle>
<button class="btn btn-xs btn-primary">Size of .btn-xs</button>
```

And here is a plunker showing the issue
https://plnkr.co/edit/ps1b0St62YSLK9e7Mu8n

**Comment by bertysentry** [comment]
Hi,

I have the very same issue... :-(

**Comment by mbacelo** [comment]
Apparently it works on previous versions (I tried in 0.1.2). Something has changed in the last version that broke sizing options

**Comment by bertysentry** [comment]
The issue is fixed in version 0.4.0. Please close the issue to confirm the fix. Thanks!
